action.yml

```diff
@@ -34,12 +34,13 @@ runs:
     - name: Download SHAs from last run
       shell: bash
       run: |
-        latest_artifact.sh "${{ inputs.github-token }}" "${{ github.repository }}" "${{ github.event.pull_request.number }}" "${{ github.head_ref }}" "${{ github.workflow }}" "dismiss-stale-approvals-shas" || true
+        latest_artifact.sh "${{ github.repository }}" "${{ github.event.pull_request.number }}" "${{ github.head_ref }}" "${{ github.workflow }}" "dismiss-stale-approvals-shas" || true
```

Comment:
Using `gh`, it needs the token in the environment. [Docs](https://github.com/withgraphite/dismiss-stale-approvals/issues/7).